src/officehours_api/backends/zoom.py

```diff
@@ -122,7 +122,7 @@ def _get_access_token(cls, user: User) -> str:
                 },
                 headers=cls._get_client_auth_headers(),
             )
-            if resp.status_code == 401:
```

Comment:
Could you please add a comment line here on why this change is needed, e.g. Zoom changed the API return status, etc.?

Reply:
Sure, if you'd like me to.

Reply:
How is this? https://github.com/tl-its-umich-edu/remote-office-hours-queue/pull/364/commits/a0596d19bf2fb08b8a7cdea8cbdc14fe6c938699

Reply:
looks good there.